media/css/firefox/whatsnew/header.scss

```diff
@@ -0,0 +1,109 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../../pebbles/includes/lib';
+
+//* -------------------------------------------------------------------------- */
+// Up-to-date header
+
+.main-header {
+    text-align: center;
+
+    .inner-container {
+        position: relative;
+    }
+
+    h1 {
+        @include font-size-level5;
+        display: none;
+        margin: 7px 0;
```

Comment:
Suggest adding `font-weight; normal;` here? (It's bold on `/id/whatsnew/`, but not on the regular `/whatsnew/` page template).

Reply:
Nice catch!